apps/sms/desktop-mock/navigator_moz_mobilemessage.js

```diff
@@ -973,6 +981,24 @@
     timestamp: now - (60000000 * 20)
   });
 
+  messagesDb.messages.push({
+    type: 'mms',
+    id: messagesDb.id++,
+    threadId: 15,
+    delivery: 'received',
+    deliveryInfo: [{
+      deliveryStatus: 'success',
+      deliveryTimestamp: 0,
+      readStatus: 'not-applicable',
+      readTimestamp: 0,
+      receiver: null
+    }],
+    sender: '+33623456789',
+    receivers: ['+33623456789','+33612345678'],
+    timestamp: now - (60000000 * 30),
+    sentTimestamp: now - (60000000 * 30)
```

Comment:
the issue you have is that the code in https://github.com/mozilla-b2g/gaia/blob/5c79eeb800161651b22af9461e3de07e527a0141/apps/sms/views/conversation/js/conversation.js#L1674 expect that `attachments` is either null or an array. Here it's `undefined` because it's not present, and as a result we have an error at that line.

I'd suggest to fix the issue by changing `message.attachments === null` to `!message.attachments` in conversations.js.